client/blocks/upsell-nudge/index.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const UpsellNudge = ( {
+	buttonText,
```

Comment:
Since we're passing down the props to the `Banner` component, we could use the rest operator:

```js
export const UpsellNudge = ( {
	buttonText,
	className,
	text,
	...props
} ) => {
	const classes = classnames( 'upsell-nudge', className );

	return (
		<Banner
			{ ...props }
			className={ classes }
			callToAction={ buttonText }
			title={ text }
		/>
	);
};
```

Also, we could simplify that further if we rename the `buttonText` and `text` props to `callToAction` and `title`.